libglusterfs/src/iobuf.c

```diff
@@ -925,6 +998,10 @@ iobuf_size(struct iobuf *iobuf)
 
     GF_VALIDATE_OR_GOTO("iobuf", iobuf, out);
 
+    if (iobuf->page_size) {
```

Comment:
Is there any case where page_size could be 0 ? I think this function is identical to `iobuf_pagesize` macro.

Reply:
done